test/integration/targets/ecs_ecr/tasks/main.yml

```diff
@@ -1,16 +1,22 @@
 ---
 - set_fact:
-    ecr_name: '{{ resource_prefix }}-ecr'
+    ecr_name: '{{ resource_prefix | lower }}-ecr'
```

Comment:
This shouldn't be necessary as the resource_prefix is only ints and already lowered values. https://github.com/ansible/ansible/blob/devel/test/lib/ansible_test/_internal/cloud/__init__.py#L398

Reply:
I think you're right. Removed.